File: cs_desktop/src/FalseCrypt.Crypto/PasswordDeriveData.cs
namespace FalseCrypt.Crypto
{
    public ref struct PasswordDeriveData
    {
        public byte[] Key { get; }

        public byte[] Salt { get; }

        public PasswordDeriveData(byte[] key, byte[] salt)
        {
            Key = key;
            Salt = salt;
        }
    }
}

File: java_android/app/src/main/java/FalseCrypt/Crypto/PasswordDeriveData.java
package FalseCrypt.Crypto;

public class PasswordDeriveData {
	public final byte[] Key;
	public final byte[] Salt;
	
	public PasswordDeriveData(final byte[] Key, final byte[] Salt) {
		this.Key = Key;
		this.Salt = Salt;
	}
}
